src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,432 @@
+		Block fenceBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "Fence", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		fenceBlock.name = name + " Fence";
+		fenceBlock.blockBase = "Fence";
+		fenceBlock.texture = planksTextureName;
+		fenceBlock.soundOnStep = new StepSound(workspace, "WOOD");
+		fenceBlock.hardness = 2 * factor;
+		fenceBlock.resistance = 3 * factor;
+		fenceBlock.breakHarvestLevel = 0;
+		fenceBlock.flammability = (int) Math.round(5 * factor);
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(fenceBlock);
+		mcreator.getWorkspace().addModElement(fenceBlock.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(fenceBlock);
+		mcreator.getWorkspace().getModElementManager().storeModElement(fenceBlock);
+
+		// we use Block GUI to get default values for the block element (kinda hacky!)
+		Block fenceGateBlock = (Block) ModElementTypeRegistry.REGISTRY.get(ModElementType.BLOCK)
+				.getModElement(mcreator, new ModElement(workspace, name + "FenceGate", ModElementType.BLOCK), false)
+				.getElementFromGUI();
+		fenceGateBlock.name = name + " Fence Gate";
+		fenceGateBlock.blockBase = "FenceGate";
+		fenceGateBlock.texture = planksTextureName;
+		fenceGateBlock.soundOnStep = new StepSound(workspace, "WOOD");
+		fenceGateBlock.hardness = 2 * factor;
+		fenceGateBlock.resistance = 3 * factor;
+		fenceGateBlock.breakHarvestLevel = 0;
+		fenceGateBlock.flammability = (int) Math.round(5 * factor);
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(fenceGateBlock);
+		mcreator.getWorkspace().addModElement(fenceGateBlock.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(fenceGateBlock);
+		mcreator.getWorkspace().getModElementManager().storeModElement(fenceGateBlock);
+
+		//Tag - Items
+		//Mainly used for recipes and loot tables
+		Tag woodItemTag = (Tag) ModElementTypeRegistry.REGISTRY.get(ModElementType.TAG)
+				.getModElement(mcreator, new ModElement(workspace, name + "ItemsTag", ModElementType.TAG), false)
+				.getElementFromGUI();
+		woodItemTag.namespace = "forge";
+		woodItemTag.name = RegistryNameFixer.fix(name) + "_log";
+		woodItemTag.type = "Items";
+		woodItemTag.items = new ArrayList<>();
+		woodItemTag.items.add(new MItemBlock(workspace, "CUSTOM:" + woodBlock.getModElement().getName()));
+		woodItemTag.items.add(new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName()));
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(woodItemTag);
+		mcreator.getWorkspace().addModElement(woodItemTag.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(woodItemTag);
+		mcreator.getWorkspace().getModElementManager().storeModElement(woodItemTag);
+
+		//Recipes
+		Recipe woodRecipe = (Recipe) ModElementTypeRegistry.REGISTRY.get(ModElementType.RECIPE)

```

Comment:
Shouldn't this use the tag just like the planks recipe does (note the namespace problem with that one implementing this here)?